{"level":"error","message":"login failed login failed","stack":"Error: login failed\n    at Object.login (C:\\Users\\mohamad\\Desktop\\auth-project\\src\\routes\\auth\\controller.js:34:11)\n    at newFn (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Object.validate (C:\\Users\\mohamad\\Desktop\\auth-project\\src\\routes\\controller.js:30:5)\n    at newFn (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at middleware (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express-validator\\src\\middlewares\\check.js:16:13)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)"}
{"level":"error","message":"something failed outside something failed outside","stack":"Error: something failed outside\n    at Object.<anonymous> (C:\\Users\\mohamad\\Desktop\\auth-project\\index.js:27:7)\n    at Module._compile (internal/modules/cjs/loader.js:1063:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)\n    at Module.load (internal/modules/cjs/loader.js:928:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:769:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)\n    at internal/main/run_main_module.js:17:47"}
{"level":"error","message":"something failed outside something failed outside","stack":"Error: something failed outside\n    at Object.<anonymous> (C:\\Users\\mohamad\\Desktop\\auth-project\\index.js:31:7)\n    at Module._compile (internal/modules/cjs/loader.js:1063:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)\n    at Module.load (internal/modules/cjs/loader.js:928:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:769:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)\n    at internal/main/run_main_module.js:17:47"}
{"level":"error","message":"something failed outside something failed outside","stack":"Error: something failed outside\n    at Object.<anonymous> (C:\\Users\\mohamad\\Desktop\\auth-project\\index.js:37:7)\n    at Module._compile (internal/modules/cjs/loader.js:1063:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)\n    at Module.load (internal/modules/cjs/loader.js:928:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:769:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)\n    at internal/main/run_main_module.js:17:47"}
{"level":"error","message":"something failed outside something failed outside","stack":"Error: something failed outside\n    at Object.<anonymous> (C:\\Users\\mohamad\\Desktop\\auth-project\\index.js:37:7)\n    at Module._compile (internal/modules/cjs/loader.js:1063:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)\n    at Module.load (internal/modules/cjs/loader.js:928:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:769:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)\n    at internal/main/run_main_module.js:17:47"}
{"level":"error","message":"something failed outside promise something failed outside promise","stack":"Error: something failed outside promise\n    at Object.<anonymous> (C:\\Users\\mohamad\\Desktop\\auth-project\\index.js:33:26)\n    at Module._compile (internal/modules/cjs/loader.js:1063:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)\n    at Module.load (internal/modules/cjs/loader.js:928:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:769:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)\n    at internal/main/run_main_module.js:17:47"}
{"level":"error","message":"login failed login failed","stack":"Error: login failed\n    at Object.login (C:\\Users\\mohamad\\Desktop\\auth-project\\src\\routes\\auth\\controller.js:34:11)\n    at Layer.handle [as handle_request] (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Object.validate (C:\\Users\\mohamad\\Desktop\\auth-project\\src\\routes\\controller.js:30:5)\n    at Layer.handle [as handle_request] (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at middleware (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express-validator\\src\\middlewares\\check.js:16:13)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)"}
{"level":"error","message":"login failed login failed","stack":"Error: login failed\n    at Object.login (C:\\Users\\mohamad\\Desktop\\auth-project\\src\\routes\\auth\\controller.js:34:11)\n    at Layer.handle [as handle_request] (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Object.validate (C:\\Users\\mohamad\\Desktop\\auth-project\\src\\routes\\controller.js:30:5)\n    at Layer.handle [as handle_request] (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at middleware (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express-validator\\src\\middlewares\\check.js:16:13)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)"}
{"level":"error","message":"login failed login failed","stack":"Error: login failed\n    at Object.login (C:\\Users\\mohamad\\Desktop\\auth-project\\src\\routes\\auth\\controller.js:34:11)\n    at Layer.handle [as handle_request] (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Object.validate (C:\\Users\\mohamad\\Desktop\\auth-project\\src\\routes\\controller.js:30:5)\n    at Layer.handle [as handle_request] (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at middleware (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express-validator\\src\\middlewares\\check.js:16:13)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)"}
{"level":"error","message":"login failed login failed","stack":"Error: login failed\n    at Object.login (C:\\Users\\mohamad\\Desktop\\auth-project\\src\\routes\\auth\\controller.js:34:11)\n    at Layer.handle [as handle_request] (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Object.validate (C:\\Users\\mohamad\\Desktop\\auth-project\\src\\routes\\controller.js:30:5)\n    at Layer.handle [as handle_request] (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at middleware (C:\\Users\\mohamad\\Desktop\\auth-project\\node_modules\\express-validator\\src\\middlewares\\check.js:16:13)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)"}
{"level":"error","message":"Operation `users.findOne()` buffering timed out after 10000ms Operation `users.findOne()` buffering timed out after 10000ms","stack":"MongooseError: Operation `users.findOne()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:149:23)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)"}
{"level":"error","message":"Operation `users.findOne()` buffering timed out after 10000ms Operation `users.findOne()` buffering timed out after 10000ms","stack":"MongooseError: Operation `users.findOne()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:149:23)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)"}
{"code":8000,"codeName":"AtlasError","level":"error","message":"user is not allowed to do action [find] on [test.users] user is not allowed to do action [find] on [test.users]","ok":0,"stack":"MongoServerError: user is not allowed to do action [find] on [test.users]\n    at MessageStream.messageHandler (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongodb\\lib\\cmap\\connection.js:467:30)\n    at MessageStream.emit (node:events:513:28)\n    at processIncomingData (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:108:16)\n    at MessageStream._write (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:28:9)\n    at writeOrBuffer (node:internal/streams/writable:392:12)\n    at _write (node:internal/streams/writable:333:10)\n    at Writable.write (node:internal/streams/writable:337:10)\n    at TLSSocket.ondata (node:internal/streams/readable:766:22)\n    at TLSSocket.emit (node:events:513:28)\n    at addChunk (node:internal/streams/readable:324:12)"}
{"code":8000,"codeName":"AtlasError","level":"error","message":"user is not allowed to do action [find] on [test.users] user is not allowed to do action [find] on [test.users]","ok":0,"stack":"MongoServerError: user is not allowed to do action [find] on [test.users]\n    at MessageStream.messageHandler (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongodb\\lib\\cmap\\connection.js:467:30)\n    at MessageStream.emit (node:events:513:28)\n    at processIncomingData (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:108:16)\n    at MessageStream._write (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:28:9)\n    at writeOrBuffer (node:internal/streams/writable:392:12)\n    at _write (node:internal/streams/writable:333:10)\n    at Writable.write (node:internal/streams/writable:337:10)\n    at TLSSocket.ondata (node:internal/streams/readable:766:22)\n    at TLSSocket.emit (node:events:513:28)\n    at addChunk (node:internal/streams/readable:324:12)"}
{"code":8000,"codeName":"AtlasError","level":"error","message":"user is not allowed to do action [find] on [test.users] user is not allowed to do action [find] on [test.users]","ok":0,"stack":"MongoServerError: user is not allowed to do action [find] on [test.users]\n    at MessageStream.messageHandler (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongodb\\lib\\cmap\\connection.js:467:30)\n    at MessageStream.emit (node:events:513:28)\n    at processIncomingData (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:108:16)\n    at MessageStream._write (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:28:9)\n    at writeOrBuffer (node:internal/streams/writable:392:12)\n    at _write (node:internal/streams/writable:333:10)\n    at Writable.write (node:internal/streams/writable:337:10)\n    at TLSSocket.ondata (node:internal/streams/readable:766:22)\n    at TLSSocket.emit (node:events:513:28)\n    at addChunk (node:internal/streams/readable:324:12)"}
{"code":8000,"codeName":"AtlasError","level":"error","message":"user is not allowed to do action [find] on [hamshab.users] user is not allowed to do action [find] on [hamshab.users]","ok":0,"stack":"MongoServerError: user is not allowed to do action [find] on [hamshab.users]\n    at MessageStream.messageHandler (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongodb\\lib\\cmap\\connection.js:467:30)\n    at MessageStream.emit (node:events:513:28)\n    at processIncomingData (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:108:16)\n    at MessageStream._write (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:28:9)\n    at writeOrBuffer (node:internal/streams/writable:392:12)\n    at _write (node:internal/streams/writable:333:10)\n    at Writable.write (node:internal/streams/writable:337:10)\n    at TLSSocket.ondata (node:internal/streams/readable:766:22)\n    at TLSSocket.emit (node:events:513:28)\n    at addChunk (node:internal/streams/readable:324:12)"}
{"level":"error","message":"Operation `users.findOne()` buffering timed out after 10000ms Operation `users.findOne()` buffering timed out after 10000ms","stack":"MongooseError: Operation `users.findOne()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:149:23)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)"}
{"level":"error","message":"Operation `users.findOne()` buffering timed out after 10000ms Operation `users.findOne()` buffering timed out after 10000ms","stack":"MongooseError: Operation `users.findOne()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:149:23)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)"}
{"level":"error","message":"Operation `users.findOne()` buffering timed out after 10000ms Operation `users.findOne()` buffering timed out after 10000ms","stack":"MongooseError: Operation `users.findOne()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:149:23)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)"}
{"level":"error","message":"Operation `users.findOne()` buffering timed out after 10000ms Operation `users.findOne()` buffering timed out after 10000ms","stack":"MongooseError: Operation `users.findOne()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:149:23)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)"}
{"level":"error","message":"Operation `users.findOne()` buffering timed out after 10000ms Operation `users.findOne()` buffering timed out after 10000ms","stack":"MongooseError: Operation `users.findOne()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:149:23)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)"}
{"level":"error","message":"Operation `users.findOne()` buffering timed out after 10000ms Operation `users.findOne()` buffering timed out after 10000ms","stack":"MongooseError: Operation `users.findOne()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:149:23)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)"}
{"level":"error","message":"Operation `users.findOne()` buffering timed out after 10000ms Operation `users.findOne()` buffering timed out after 10000ms","stack":"MongooseError: Operation `users.findOne()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:149:23)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)"}
{"_message":"User validation failed","errors":{"first_name":{"kind":"required","message":"Path `first_name` is required.","name":"ValidatorError","path":"first_name","properties":{"message":"Path `first_name` is required.","path":"first_name","type":"required"}},"last_name":{"kind":"required","message":"Path `last_name` is required.","name":"ValidatorError","path":"last_name","properties":{"message":"Path `last_name` is required.","path":"last_name","type":"required"}}},"level":"error","message":"User validation failed: last_name: Path `last_name` is required., first_name: Path `first_name` is required. User validation failed: last_name: Path `last_name` is required., first_name: Path `first_name` is required.","stack":"ValidationError: User validation failed: last_name: Path `last_name` is required., first_name: Path `first_name` is required.\n    at Document.invalidate (C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongoose\\lib\\document.js:2907:32)\n    at C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongoose\\lib\\document.js:2698:17\n    at C:\\Users\\EasyNet\\Desktop\\auth-project\\node_modules\\mongoose\\lib\\schematype.js:1280:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"}
{"level":"error","message":"this.User is not a constructor this.User is not a constructor","stack":"TypeError: this.User is not a constructor\n    at module.exports.create (C:\\Users\\EasyNet\\Desktop\\backend\\src\\routes\\pool\\controller.js:8:16)\n    at Layer.handle [as handle_request] (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\express\\lib\\router\\route.js:144:13)\n    at module.exports.validate (C:\\Users\\EasyNet\\Desktop\\backend\\src\\routes\\controller.js:30:5)\n    at Layer.handle [as handle_request] (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\express\\lib\\router\\route.js:144:13)\n    at middleware (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\express-validator\\src\\middlewares\\check.js:16:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Operation `pools.insertOne()` buffering timed out after 10000ms Operation `pools.insertOne()` buffering timed out after 10000ms","stack":"MongooseError: Operation `pools.insertOne()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:185:23)\n    at listOnTimeout (node:internal/timers:569:17)\n    at process.processTimers (node:internal/timers:512:7)"}
{"code":11000,"index":0,"keyPattern":{"title":1},"keyValue":{"title":"text1"},"level":"error","message":"E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"text1\" } E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"text1\" }","stack":"MongoServerError: E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"text1\" }\n    at C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\operations\\insert.js:50:33\n    at C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:331:21\n    at C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\sdam\\server.js:207:17\n    at handleOperationResult (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\sdam\\server.js:335:20)\n    at Connection.onMessage (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:206:9)\n    at MessageStream.<anonymous> (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:61:60)\n    at MessageStream.emit (node:events:513:28)\n    at processIncomingData (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:124:16)\n    at MessageStream._write (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:33:9)\n    at writeOrBuffer (node:internal/streams/writable:392:12)"}
{"code":11000,"index":0,"keyPattern":{"title":1},"keyValue":{"title":"text1"},"level":"error","message":"E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"text1\" } E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"text1\" }","stack":"MongoServerError: E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"text1\" }\n    at C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\operations\\insert.js:50:33\n    at C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:331:21\n    at C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\sdam\\server.js:207:17\n    at handleOperationResult (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\sdam\\server.js:335:20)\n    at Connection.onMessage (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:206:9)\n    at MessageStream.<anonymous> (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:61:60)\n    at MessageStream.emit (node:events:513:28)\n    at processIncomingData (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:124:16)\n    at MessageStream._write (C:\\Users\\EasyNet\\Desktop\\backend\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:33:9)\n    at writeOrBuffer (node:internal/streams/writable:392:12)"}
{"_message":"Pool validation failed","errors":{"ticket_value":{"kind":"Number","message":"Cast to Number failed for value \"asd\" (type string) at path \"ticket_value\"","name":"CastError","path":"ticket_value","reason":{"actual":false,"code":"ERR_ASSERTION","expected":true,"generatedMessage":true,"operator":"=="},"stringValue":"\"asd\"","value":"asd","valueType":"string"}},"level":"error","message":"Pool validation failed: ticket_value: Cast to Number failed for value \"asd\" (type string) at path \"ticket_value\" Pool validation failed: ticket_value: Cast to Number failed for value \"asd\" (type string) at path \"ticket_value\"","stack":"ValidationError: Pool validation failed: ticket_value: Cast to Number failed for value \"asd\" (type string) at path \"ticket_value\"\n    at model.Document.invalidate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:3163:32)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1460:12)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1120:16)\n    at model.Document (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:165:12)\n    at model.Model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:122:12)\n    at new model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:4653:15)\n    at Object.create (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\pool\\controller.js:8:16)\n    at Layer.handle [as handle_request] (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\route.js:144:13)\n    at Object.validate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\controller.js:30:5)"}
{"_message":"Pool validation failed","errors":{"ticket_value":{"kind":"Number","message":"Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"","name":"CastError","path":"ticket_value","reason":{"actual":false,"code":"ERR_ASSERTION","expected":true,"generatedMessage":true,"operator":"=="},"stringValue":"\"sdfsd\"","value":"sdfsd","valueType":"string"}},"level":"error","message":"Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\" Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"","stack":"ValidationError: Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"\n    at model.Document.invalidate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:3163:32)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1460:12)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1120:16)\n    at model.Document (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:165:12)\n    at model.Model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:122:12)\n    at new model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:4653:15)\n    at Object.create (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\pool\\controller.js:8:16)\n    at Layer.handle [as handle_request] (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\route.js:144:13)\n    at Object.validate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\controller.js:30:5)"}
{"_message":"Pool validation failed","errors":{"ticket_value":{"kind":"Number","message":"Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"","name":"CastError","path":"ticket_value","reason":{"actual":false,"code":"ERR_ASSERTION","expected":true,"generatedMessage":true,"operator":"=="},"stringValue":"\"sdfsd\"","value":"sdfsd","valueType":"string"}},"level":"error","message":"Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\" Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"","stack":"ValidationError: Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"\n    at model.Document.invalidate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:3163:32)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1460:12)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1120:16)\n    at model.Document (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:165:12)\n    at model.Model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:122:12)\n    at new model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:4653:15)\n    at Object.create (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\pool\\controller.js:8:16)\n    at Layer.handle [as handle_request] (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\route.js:144:13)\n    at Object.validate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\controller.js:30:5)"}
{"_message":"Pool validation failed","errors":{"ticket_value":{"kind":"Number","message":"Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"","name":"CastError","path":"ticket_value","reason":{"actual":false,"code":"ERR_ASSERTION","expected":true,"generatedMessage":true,"operator":"=="},"stringValue":"\"sdfsd\"","value":"sdfsd","valueType":"string"}},"level":"error","message":"Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\" Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"","stack":"ValidationError: Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"\n    at model.Document.invalidate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:3163:32)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1460:12)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1120:16)\n    at model.Document (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:165:12)\n    at model.Model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:122:12)\n    at new model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:4653:15)\n    at Object.create (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\pool\\controller.js:8:16)\n    at Layer.handle [as handle_request] (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\route.js:144:13)\n    at Object.validate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\controller.js:30:5)"}
{"_message":"Pool validation failed","errors":{"ticket_value":{"kind":"Number","message":"Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"","name":"CastError","path":"ticket_value","reason":{"actual":false,"code":"ERR_ASSERTION","expected":true,"generatedMessage":true,"operator":"=="},"stringValue":"\"sdfsd\"","value":"sdfsd","valueType":"string"}},"level":"error","message":"Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\" Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"","stack":"ValidationError: Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"\n    at model.Document.invalidate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:3163:32)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1460:12)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1120:16)\n    at model.Document (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:165:12)\n    at model.Model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:122:12)\n    at new model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:4653:15)\n    at Object.create (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\pool\\controller.js:8:16)\n    at Layer.handle [as handle_request] (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\route.js:144:13)\n    at Object.validate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\controller.js:30:5)"}
{"_message":"Pool validation failed","errors":{"ticket_value":{"kind":"Number","message":"Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"","name":"CastError","path":"ticket_value","reason":{"actual":false,"code":"ERR_ASSERTION","expected":true,"generatedMessage":true,"operator":"=="},"stringValue":"\"sdfsd\"","value":"sdfsd","valueType":"string"}},"level":"error","message":"Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\" Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"","stack":"ValidationError: Pool validation failed: ticket_value: Cast to Number failed for value \"sdfsd\" (type string) at path \"ticket_value\"\n    at model.Document.invalidate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:3163:32)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1460:12)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1120:16)\n    at model.Document (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:165:12)\n    at model.Model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:122:12)\n    at new model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:4653:15)\n    at Object.create (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\pool\\controller.js:8:16)\n    at Layer.handle [as handle_request] (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\route.js:144:13)\n    at Object.validate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\controller.js:30:5)"}
{"code":11000,"index":0,"keyPattern":{"title":1},"keyValue":{"title":"Sina"},"level":"error","message":"E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"Sina\" } E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"Sina\" }","stack":"MongoServerError: E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"Sina\" }\n    at C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\operations\\insert.js:50:33\n    at C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:331:21\n    at C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\sdam\\server.js:207:17\n    at handleOperationResult (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\sdam\\server.js:335:20)\n    at Connection.onMessage (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:206:9)\n    at MessageStream.<anonymous> (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:61:60)\n    at MessageStream.emit (node:events:513:28)\n    at processIncomingData (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:124:16)\n    at MessageStream._write (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:33:9)\n    at writeOrBuffer (node:internal/streams/writable:391:12)"}
{"code":11000,"index":0,"keyPattern":{"title":1},"keyValue":{"title":"Sina"},"level":"error","message":"E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"Sina\" } E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"Sina\" }","stack":"MongoServerError: E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"Sina\" }\n    at C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\operations\\insert.js:50:33\n    at C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:331:21\n    at C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\sdam\\server.js:207:17\n    at handleOperationResult (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\sdam\\server.js:335:20)\n    at Connection.onMessage (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:206:9)\n    at MessageStream.<anonymous> (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:61:60)\n    at MessageStream.emit (node:events:513:28)\n    at processIncomingData (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:124:16)\n    at MessageStream._write (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:33:9)\n    at writeOrBuffer (node:internal/streams/writable:391:12)"}
{"code":11000,"index":0,"keyPattern":{"title":1},"keyValue":{"title":"Sina"},"level":"error","message":"E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"Sina\" } E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"Sina\" }","stack":"MongoServerError: E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"Sina\" }\n    at C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\operations\\insert.js:50:33\n    at C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:331:21\n    at C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\sdam\\server.js:207:17\n    at handleOperationResult (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\sdam\\server.js:335:20)\n    at Connection.onMessage (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:206:9)\n    at MessageStream.<anonymous> (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:61:60)\n    at MessageStream.emit (node:events:513:28)\n    at processIncomingData (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:124:16)\n    at MessageStream._write (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:33:9)\n    at writeOrBuffer (node:internal/streams/writable:391:12)"}
{"_message":"Pool validation failed","errors":{"ticket_value":{"kind":"Number","message":"Cast to Number failed for value \"sdfs\" (type string) at path \"ticket_value\"","name":"CastError","path":"ticket_value","reason":{"actual":false,"code":"ERR_ASSERTION","expected":true,"generatedMessage":true,"operator":"=="},"stringValue":"\"sdfs\"","value":"sdfs","valueType":"string"}},"level":"error","message":"Pool validation failed: ticket_value: Cast to Number failed for value \"sdfs\" (type string) at path \"ticket_value\" Pool validation failed: ticket_value: Cast to Number failed for value \"sdfs\" (type string) at path \"ticket_value\"","stack":"ValidationError: Pool validation failed: ticket_value: Cast to Number failed for value \"sdfs\" (type string) at path \"ticket_value\"\n    at model.Document.invalidate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:3163:32)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1460:12)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1120:16)\n    at model.Document (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:165:12)\n    at model.Model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:122:12)\n    at new model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:4653:15)\n    at Object.create (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\pool\\controller.js:8:16)\n    at Layer.handle [as handle_request] (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\route.js:144:13)\n    at Object.validate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\controller.js:30:5)"}
{"_message":"Pool validation failed","errors":{"ticket_value":{"kind":"Number","message":"Cast to Number failed for value \"sdfs\" (type string) at path \"ticket_value\"","name":"CastError","path":"ticket_value","reason":{"actual":false,"code":"ERR_ASSERTION","expected":true,"generatedMessage":true,"operator":"=="},"stringValue":"\"sdfs\"","value":"sdfs","valueType":"string"}},"level":"error","message":"Pool validation failed: ticket_value: Cast to Number failed for value \"sdfs\" (type string) at path \"ticket_value\" Pool validation failed: ticket_value: Cast to Number failed for value \"sdfs\" (type string) at path \"ticket_value\"","stack":"ValidationError: Pool validation failed: ticket_value: Cast to Number failed for value \"sdfs\" (type string) at path \"ticket_value\"\n    at model.Document.invalidate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:3163:32)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1460:12)\n    at model.$set (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:1120:16)\n    at model.Document (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\document.js:165:12)\n    at model.Model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:122:12)\n    at new model (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongoose\\lib\\model.js:4653:15)\n    at Object.create (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\pool\\controller.js:8:16)\n    at Layer.handle [as handle_request] (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\express\\lib\\router\\route.js:144:13)\n    at Object.validate (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\src\\routes\\controller.js:30:5)"}
{"code":11000,"index":0,"keyPattern":{"title":1},"keyValue":{"title":"Sinafaafsdafasd"},"level":"error","message":"E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"Sinafaafsdafasd\" } E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"Sinafaafsdafasd\" }","stack":"MongoServerError: E11000 duplicate key error collection: nasim_test.pools index: title_1 dup key: { title: \"Sinafaafsdafasd\" }\n    at C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\operations\\insert.js:50:33\n    at C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:331:21\n    at C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\sdam\\server.js:207:17\n    at handleOperationResult (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\sdam\\server.js:335:20)\n    at Connection.onMessage (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:206:9)\n    at MessageStream.<anonymous> (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\connection.js:61:60)\n    at MessageStream.emit (node:events:513:28)\n    at processIncomingData (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:124:16)\n    at MessageStream._write (C:\\Users\\Sina\\OneDrive\\Desktop\\lucky_duck\\backend\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:33:9)\n    at writeOrBuffer (node:internal/streams/writable:391:12)"}
